ui/components/policy/req.js

```diff
@@ -1,15 +1,40 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 
+import { filterAppliesTo } from '../requirements/requirement';
+import Metadata from '../requirements/metadata';
+import TopicLink from '../requirements/topic-link';
+
 export default function Req({ highlighted, href, onClick, req }) {
   let meta = null;
   if (highlighted) {
     meta = (
-      <div className="col col-4">
-        Number: { req.req_id }
-        <ul>
-          { req.topics.map(t => <li key={t.id}>{t.name}</li>) }
-        </ul>
+      <div className="req col col-4">
+        <Metadata className="requirement-id" name="Requirement ID" value={req.req_id} />
+        <Metadata
+          className="issuance"
+          name="Policy issuance"
+          value={req.policy.issuance}
+        />
```

Comment:
We should probably remove the `policy` fields as they apply to the whole document rather than individual requirements.